<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupDialog</class>
 <widget class="QDialog" name="SignupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="signPushButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>311</width>
     <height>162</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idLineEdit">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入您的用户名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameLineEdit">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="placeholderText">
       <string>请输入您的姓名</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="pwdLineEdit">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请输入您的密码</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>再次输入密码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="repwdLineEdit">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>请再次输入您的密码</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="managerCheckBox">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>申请管理员权限</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>SignupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
